src/overview/components/Overview.jsx

```diff
@@ -75,6 +91,9 @@ const mapDispatchToProps = dispatch => ({
     onEndDateChange: date => {
         dispatch(actions.setEndDate({endDate: date}))
     },
+    getMoreResults: (date, position) => {
+        dispatch(actions.getMoreResults({loadingIndicator: true, endDate: date, scrollPosition: position}))
```

Comment:
loadingIndicator seems a mix of business logic, why not letting the action/reducer decide and use something like "fetching"?